action/edit.php

```diff
@@ -34,17 +35,49 @@ public function register(Doku_Event_Handler $controller)
     {
         // add the struct editor to the edit form;
         $controller->register_hook('HTML_EDITFORM_OUTPUT', 'BEFORE', $this, 'handleEditform');
+        $controller->register_hook('FORM_EDIT_OUTPUT', 'BEFORE', $this, 'addFromData');
+    }
+
+    /**
+     * Adds the html for the struct editors to the edit from
+     *
+     * Handles the FORM_EDIT_OUTPUT event
+     */
+    public function addFromData(Doku_Event $event, $_param): bool
```

Comment:
This declaration should be changed to retain compatibility with PHP 5.6, still officially supported by DokuWiki.

Reply:
Hogfather _still_ supports PHP 5.6? 🧐

Fixed.